resources/styles/components/scores/index.less

```diff
@@ -162,17 +162,20 @@
       display: inline-block;
       width: 50%;
       text-align: center;
+      overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+      
       &:first-child {
         border-right: 1px solid @nav-tabs-border-color;
       }
     }
   }
-  .name-cell .name {
-    padding-left: 1rem;
-    text-align: left;
-    display: inline-block;
```

Comment:
The `text-align: left;` rule here was used to remove the default center alignment for the name column.  I think we probably want to retain that :grin: